src/rantanplan/rhymes.py

```diff
@@ -32,7 +32,6 @@
     ("v", "b"), ("ll", "y"),
     ("ze", "ce"), ("zi", "ci"),
     ("qui", "ki"), ("que", "ke"),
-    ("ge", "je"), ("gi", "ji")
```

Comment:
My previous comment was on the trailing comma, that appeared then disappeared. That line needs to be there:
```
    ("ge", "je"), ("gi", "ji"),
```

Reply:
I have to do more testing before adding that line anyways